### DISCUSSION TITLE
Update Opentelemetry.Extensions to using  OpenTelemetry 1.4.0-beta.3

### DISCUSSION THREAD

**Discussion by michalwawa** [open-telemetry/opentelemetry-dotnet#3988] [Ideas]
Hi

Right now Opentelemetry.Extensions is using OpenTelemetry.1.4.0-beta.2 and we are not able to use it with OpenTelemetry.1.4.0.beta.3.

The only solution is to downgrade all libraries to use OpenTelemetry.1.4.0-beta.2.

Is there any ETA to upgrade Opentelemetry.Extensions to using OpenTelemetry.1.4.0.beta.3 ?

Thanks

**Comment by Kielek** [comment]
All changes are in place, could you please create PR to https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Extensions/CHANGELOG.md
based on https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/CONTRIBUTING.md#how-to-request-for-release-of-package ?

One of the latest similar PRs is https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/806
